source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -76,19 +77,49 @@ Finalization
+
+Finalization happens at a minimum one second after block creation. A new block has to be created descended from that block for finalization to happen.
+
 When a sufficiently large number of members of the committee have received the block and agree on its outcome, the block is finalized. Newer blocks must have the finalized block as an ancestor to ensure the integrity of the chain.
 
 Finalization committee
 ----------------------
 
-The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total amount of existing CCD. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold.
+The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total stake of CCD in pools. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold. There is a minimum and maximum size for the finalization committee; it is 0.1% of the stake in pools, but always at least x bakers and always at most y bakers (e.g., x = 25 and y = 500).
 
 Participating in the finalization committee produces rewards on each block that is finalized. The rewards are paid to the baker account some time after the block is finalized.
 
 Overview of the baker process
 =============================
 
+#. A Proof of stake based lottery system (verifiable random function) produces a list of bakers. The higher the baker’s stake, the higher the probability of being included on the list more often.
+
+#. The first baker on the list (Bob) makes a new block that appends to the genesis block.
+
+#. Bob then broadcasts the block to the finalizers.
+
+#. If the block is valid, the finalizers will sign it.
+
+#. If the combined stake of the finalizers who sign the block is greater than two-thirds of the total stake, the block gets a :ref:`Quorum Certificate (QC)<glossary-quorum-certificate>` that certifies that this is a valid block.
+
+.. image:: ../images/concepts/baker-process1.png
+   :alt: diagram of baker process
+
+6. The next baker (Alice) now needs the QC to produce the next block. The new block can only extend the previous block whose QC is presented to Alice, so fork formation is not possible.
```

Comment:
Not only will Alice need it. But in particular Alice will never progress to the new round before she sees the QC.

Reply:
I handled this in a slightly different way, by adding it to the previous step.